conductr_cli/bndl_main.py

```diff
@@ -364,6 +369,19 @@ def add_conf_arguments(parser):
                              'Example: bndl --volume my-vol=/data --component web',
                         metavar='NAME=MOUNT-POINT')
 
+    parser.add_argument('--with-defaults',
+                        default=None,
```

Comment:
How do you feel about specifying the default of `generic` here (in the parser) instead of checking in the actual program code?